libs/core/mappers/sorting-params.mapper.ts

```diff
@@ -0,0 +1,25 @@
+import { ActiveFieldDto, SortingParamsDto } from '../dtos/sorting-params.dto';
+import { ActiveField, SortingParams } from '../models/sorting-params';
+
+export namespace SortingParamsMapper {
+
+	/**
+	 * Maps model to dto.
+	 * @param model SortingParams model.
+	 */
+	export function toDto(model: SortingParams): SortingParamsDto {
+		const direction = model.direction === 'desc' ? '-' : '';
```

Comment:
```suggestion
		const direction = model.direction === Direction.Descending ? '-' : '';
```
Let's use `Direction`